### ISSUE TITLE
Rotating spherical harmonics gif not showing up

### ISSUE THREAD

**Issue by zmoon** [milankl/SpeedyWeather.jl#96]
<https://milankl.github.io/SpeedyWeather.jl/dev/spectral_transform/>

![image](https://user-images.githubusercontent.com/15079414/176220468-21c11fd1-415c-480f-9253-ed8a83c73629.png)

Added in #89

---

There is <https://upload.wikimedia.org/wikipedia/commons/1/12/Rotating_spherical_harmonics.gif> available?

```
![Rotating spherical harmonics](https://upload.wikimedia.org/wikipedia/commons/1/12/Rotating_spherical_harmonics.gif)
```
![Rotating spherical harmonics](https://upload.wikimedia.org/wikipedia/commons/1/12/Rotating_spherical_harmonics.gif)

**Comment by milankl** [comment]
Hmmm I wanted to use `/docs/img/Rotating_spherical_harmonics.gif` (which I copied from there into the repository) to have the picture within the repo, but I guess we could also just use an external link...

**Comment by milankl** [comment]
Closed with #109
